### DISCUSSION TITLE
Kubectl exec is not working after upgrading the Istio to 1.22.4

### DISCUSSION THREAD

**Discussion by smoharana1977** [istio/istio#53316] [General]
Hi, I am getting the below error when running the exec command on Ingress pod.

Command:
 kubectl exec -it istio-ingressgateway-7f4cc575dd-2ctf2  -n istio-system -- /bin/bash 

Error:
error: Internal error occurred: error executing command in container: failed to exec in container: failed to start exec "67f240e665255786217381b5cb52708c3a45c238365b5ec533874f451bff24a6": OCI runtime exec failed: exec failed: unable to start container process: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown

Can you please advise? I was able to exec in Istio version 1.18.2

**Comment by ilrudie** [comment]
the image used probably changed to the distroless variant.

**Reply by smoharana1977** [reply]
how do we get into the pod for distroless image?

**Reply by howardjohn** [reply]
https://istio.io/latest/docs/ops/configuration/security/harden-docker-images/#debugging

**Reply by smoharana1977** [reply]
thank you! Able to get the debug prompt. But the mounted files are unable to access. Like /etc/istio/

How do I access the mounted files from Istio ingress?

**Reply by howardjohn** [reply]
I think the `kubectl debug` command cannot include volume mounts, but technically you can send a raw patch with an ephemeral container... a bit of a pain though.

Maybe easiest is to just disable distroless image for that workload. The link above shows steps on how to enable it, you can just do the same but use `debug` instead of `disrtoless`
